pages/docs/reference/using-gradle.md

```diff
@@ -252,72 +357,128 @@ dependencies {
 ```
 
 </div>
+</div>
+
+<div class="multi-language-sample" data-lang="kotlin">
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
+
+```kotlin
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile(kotlin("stdlib"))
+}
+```
 
-If you target JavaScript, use `compile "org.jetbrains.kotlin:kotlin-stdlib-js"` instead.
+</div>
+</div>
 
-If you're targeting JDK 7 or JDK 8, you can use extended versions of the Kotlin standard library which contain
-additional extension functions for APIs added in new JDK versions. Instead of `kotlin-stdlib`, use one of the
+To run applications that use `kotlin-stdlib`, JDK 6 is required. On JDK 7, 8, or later, you can use extended versions of the Kotlin standard library which contain
```

Comment:
In fact, running those application requires a JRE, not JDK (the latter is used for building but is not required for running a Java application). Also, formally, the Android runtime is nether a JRE nor a JDK.
Using `kotlin-stdlib` with Java 7, 8 and above is fine, so saying that JDK 6 is required may be confusing.
Consider rewording this to something like:
> The Kotlin standard library `kotlin-stdlib` targets Java 6 and above. There are extended versions of the standard library that add support for some of the features of JDK 7 and JDK 8.

Reply:
+